test/python/algorithms/optimizers/test_nlopt_optimizers.py

```diff
@@ -1,60 +0,0 @@
-# This code is part of Qiskit.
-#
-# (C) Copyright IBM 2018, 2020.
-#
-# This code is licensed under the Apache License, Version 2.0. You may
-# obtain a copy of this license in the LICENSE.txt file in the root directory
-# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
-#
-# Any modifications or derivative works of this code must retain this
-# copyright notice, and modified files need to carry a notice indicating
-# that they have been altered from the originals.
-
-""" Test NLOpt Optimizers """
-
-import unittest
-from test.python.algorithms import QiskitAlgorithmsTestCase
-from ddt import ddt, idata, unpack
-from scipy.optimize import rosen
-import numpy as np
-
-from qiskit.exceptions import MissingOptionalLibraryError
-from qiskit.algorithms.optimizers import CRS, DIRECT_L, DIRECT_L_RAND
-
-
-@ddt
-class TestNLOptOptimizers(QiskitAlgorithmsTestCase):
-    """Test NLOpt Optimizers"""
-
-    def _optimize(self, optimizer, use_bound):
-        x_0 = [1.3, 0.7, 0.8, 1.9, 1.2]
-        bounds = [(-6, 6)] * len(x_0) if use_bound else None
-        res = optimizer.optimize(len(x_0), rosen, initial_point=x_0, variable_bounds=bounds)
-        np.testing.assert_array_almost_equal(res[0], [1.0] * len(x_0), decimal=2)
-        return res
-
-    # ESCH and ISRES do not do well with rosen
```

Comment:
I guess you merged this in with the other optimzers - guess that's fine. I will note that the comment here was noting that two of the nlopt optimizers ESCH and ISRES were not in the list since they did not work well with rosen. Whether something else could be find I am not sure - either way it explained their absence and this observation has been lost with the deletion

Reply:
Ah, good point I'll add the comment again!